decapode/config.py

```diff
@@ -24,3 +25,13 @@
 SINCE = '1w'
 # seconds to wait for between batches
 SLEEP_BETWEEN_BATCHES = 60
+
+# -- Kafka integration config -- #
+ENABLE_KAFKA = True
```

Comment:
I do not understand the need of this variable. Decapode should be running with or without kafka enabled.

Reply:
The aim is to be able to choose whenever we use Decapode whether we want to simply crawl resources or also produce Kafka messages. Do you think I should just remove it and count on the fact that Decapode will never be used without Kafka?